src/app.ts

```diff
@@ -62,6 +64,13 @@ const cookieConfig = {
 const sessionStore = new SessionStore(new Redis(`redis://${config.CACHE_SERVER}`));
 app.use(SessionMiddleware(cookieConfig, sessionStore));
 
+const csrfProtectionMiddleware = CsrfProtectionMiddleware({
+  sessionStore,
+  enabled: false,
```

Comment:
Probably worth adding a TODO comment in the code at this point to make clear that this middleware will be enabled when all journeys implemented (which I think is the case).